CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
 ## [v3.0.0](https://github.com/voxpupuli/puppet-windowsfeature/tree/v3.0.0) (2017-09-17)
-[Full Changelog](https://github.com/voxpupuli/puppet-windowsfeature/compare/v0.0.3...v3.0.0)
+[Full Changelog](https://github.com/voxpupuli/puppet-windowsfeature/compare/v2.1.0...v3.0.0)
```

Comment:
Maybe test or check with @alexjfisher but I don't think this change is needed - the changelog generator should pull in the stuff from `HISTORY.md`

Reply:
I ran the changelog generator after i created the history.md on this branch and it fixed the version diff.  last release it was wrong since there was no history.md and the changelog has an incorrect version format in the changelog

Reply:
ah got it.

Reply:
Yep.  This is expected now that you're not affected by https://github.com/skywinder/github-changelog-generator/issues/555 anymore.